src/bitcoin-cli.cpp

```diff
@@ -1170,6 +1170,14 @@ static int CommandLineRPC(int argc, char *argv[])
                 fputc('\n', stdout);
             }
         }
+        int nRh = 0;
+        if (gArgs.IsArgSet("-getinfo")) nRh++;
+        if (gArgs.GetBoolArg("-netinfo", false)) nRh++;
+        if (gArgs.GetBoolArg("-generate", false)) nRh++;
+        if (gArgs.GetBoolArg("-addrinfo", false)) nRh++;
+        if (nRh > 1) {
```

Comment:
```suggestion
        if (gArgs.IsArgSet("-getinfo") + gArgs.GetBoolArg("-netinfo", false) + gArgs.GetBoolArg("-generate", false) + gArgs.GetBoolArg("-addrinfo", false) > 1) {
```

This is admittedly a bit obscure, so maybe there's a better way (but it's still an improvement over the temporary variable IMO)